chain/client/src/client.rs

```diff
@@ -1927,12 +1927,6 @@ impl Client {
     // Returns detailed information about the upcoming blocks in the form of printables.
     pub fn detailed_upcoming_blocks_info_as_printable(&self) -> Result<String, Error> {
         let height_status_map = self.detailed_upcoming_blocks_info()?;
-        let use_colour = matches!(self.config.log_summary_style, LogSummaryStyle::Colored);
```

Comment:
Sorry, the Issue wasn’t clear.  We still want to preserve the colouring if colours are enabled in the log.  We just don’t want it to depend on the --log-summary-style flag but on the --color flag.